flask_app/controllers/tasks.py

```diff
@@ -1,4 +1,4 @@
-from crypt import methods
```

Comment:
@aacole9 I'm thinking you were going to add the bcyrpt library here but not sure we need it on non user tables.